docs/CODERABBIT_SETUP.md

```
@@ -0,0 +1,174 @@
+CodeRabbit provides AI-powered code reviews with support for:
+
+- Automatic docstring generation
+- Unit test generation (early access)
+- Comprehensive code analysis
+- Security scanning
+- Multi-language support
+
+## Table of Contents
+
+- [Configuration Files](#configuration-files)
+  - [1. `.coderabbit.yaml` - Main Configuration](#1-coderabbitayaml---main-configuration)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)